frontend/src/metabase/ui/components/inputs/Checkbox/Checkbox.styled.tsx

```diff
@@ -115,7 +115,7 @@ export const getCheckboxOverrides = (): MantineThemeOverride["components"] => ({
           },
         },
         labelWrapper: {
-          psoition: "relative",
+          position: "relative",
```

Comment:
since the previous change has a typo, have you looked at the component if it behaves differently now?